### PR TITLE
Add a news article for after the league

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This aims to capture and build on the excitement, without revealing any details of who may have actually won any of the awards (even though we know that some of them are publicly obvious).

Contributes towards https://github.com/srobo/tasks/issues/412

### CODE REVIEW

File: _posts/2020-07-19-stage-set-for-knockouts.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+After two weekends and 48 matches, the league stage of the SR2020 Virtual
+Competition is complete, setting the stage for the knockouts and the grand final
+next weekend.
+
+This weekend saw strong performances from a number of teams, including a number
+of firsts: Kenilworth Sixth Form collected 5 tokens into their scoring zone in
+one match and both Hills Road Sixth Form College's first team and JAMDynamics
+stealing tokens from each others scoring zones.
```
Comment (PeterJCLaw) [inline _posts/2020-07-19-stage-set-for-knockouts.md:13]: If the livestream is ready soon, might be nice to link to the specific timings of these events.
Not a blocker though -- it's a small thing and we can add it later if we want to .

File: _posts/2020-07-19-stage-set-for-knockouts.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+title: The stage is set for the SR2020 Knockouts
+---
+
+After two weekends and 48 matches, the league stage of the SR2020 Virtual
+Competition is complete, setting the stage for the knockouts and the grand final
+next weekend.
+
+This weekend saw strong performances from a number of teams, including a number
+of firsts: Kenilworth Sixth Form collected 5 tokens into their scoring zone in
+one match and both Hills Road Sixth Form College's first team and JAMDynamics
```
Comment (RealOrangeOne) [inline _posts/2020-07-19-stage-set-for-knockouts.md:12]: Was actually HRS2 in the steal

```suggestion
one match and both Hills Road Sixth Form College's second team and JAMDynamics
```

File: _posts/2020-07-19-stage-set-for-knockouts.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+for a number of other awards, including the Committee Award (given to the team
+that displays the most extraordinary ingenuity in their robot), and the Online
+Presence Award (for the team that is judged to have the best online presence).
+Full details of these awards and those for earlier stages of the competition are
+available in our [rulebook](/docs/resources/2020/rulebook.pdf)
+
+If you would like to catch up on this weekend's livestreamed matches, they're
+available on YouTube:
+
+* [Saturday: League Session 1](https://youtu.be/symUE1E4niI)
```
Comment (RealOrangeOne) [inline _posts/2020-07-19-stage-set-for-knockouts.md:36]: Looks like the link text needs updating here.

Comment (PeterJCLaw) [reply _posts/2020-07-19-stage-set-for-knockouts.md]: 625e15d reworks this somewhat.

Comment (PeterJCLaw) [reply _posts/2020-07-19-stage-set-for-knockouts.md]: 625e15d reworks this somewhat.

File: _posts/2020-07-19-stage-set-for-knockouts.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+If you would like to catch up on any of the league matches from the last two
+weekends, they're available on YouTube:
+
+| Date          | Competition Stage                                |
+|---------------|--------------------------------------------------|
+| Saturday 11th | [League Session 1](https://youtu.be/xLL7SoQywf4) |
+| Sunday 12th   | [League Session 2](https://youtu.be/7JoW4zXSeZE) |
+| Saturday 18th | [League Session 1](https://youtu.be/symUE1E4niI) |
+| Sunday 19th   | [League Session 2](https://youtu.be/Y4h5P47j8jM) |
```
Comment (RealOrangeOne) [inline _posts/2020-07-19-stage-set-for-knockouts.md:41]:
```suggestion
| Saturday 18th | [League Session 3](https://youtu.be/symUE1E4niI) |
| Sunday 19th   | [League Session 4](https://youtu.be/Y4h5P47j8jM) |
```

Comment (PeterJCLaw) [reply _posts/2020-07-19-stage-set-for-knockouts.md:41]: fail.

**Pull request merged**